wood/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     api "androidx.paging:paging-rxjava2:$pagingVersion"
     api "androidx.paging:paging-runtime:$pagingVersion"
     api "androidx.lifecycle:lifecycle-extensions:$lifecycleVersion"
+    api "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleModelVersion"
```

Comment:
Without this line of code, it will break the ci with the following error message

>Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class androidx.lifecycle.ViewModelLazy found in modules jetified-lifecycle-viewmodel-ktx-2.2.0-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0) and lifecycle-viewmodel-2.5.0-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.0)

This is probably a bug tracked in https://issuetracker.google.com/issues/238425626